modules/ndu/src/backend/conditions.ts

```diff
@@ -122,5 +122,29 @@ export const dialogConditions: sdk.Condition[] = [
     evaluate: () => {
       return 1
     }
+  },
+  {
+    id: 'type_text',
+    label: 'The user typed something specific',
+    description: `The user typed {text}`,
+    params: {
+      text: { label: 'One or multiple words to detect (one per line)', type: 'array', rows: 5 },
+      partial: { label: 'Allow partial match', type: 'boolean', defaultValue: true },
```

Comment:
can you be more specific about what means partial

Reply:
How would you write that? If "text includes other text", if "text is part of othertext"